.github/workflows/url-checker.yml

```diff
@@ -1,22 +1,58 @@
 name: Check URL Status
 
 on:
+  workflow_dispatch:
   schedule:
     - cron: 0 0 * * 0
+env:
+  LINK_TRACKING_ISSUE_NUMBER: 322
+  LYCHEE_REPORT_FILE: ./lychee/out.md
+permissions:
+  issues: write
 
 jobs:
-  check_urls:
+  linkChecker:
```

Comment:
does the mixed casing matter on this repo?
file contains snake_case, camelCase and kebab-case

Reply:
Not at all. A remnant of ChatGPT that I was a bit too lazy to fix 😁. Fixed now